### DISCUSSION TITLE
Lower required Android version

### DISCUSSION THREAD

**Discussion by rumly111** [ihhub/fheroes2#6279] [Q&A] [answered]
Is it possible to lower required Android version to 4.0.4?
Is there any technical reason not to do it?

**Comment by zenseii** [comment, marked as answer]
Hi, @rumly111.

Unfortunately it is for a technical reason, not because we decided on a random version. Naturally we'd want it to support as many versions as reasonably possible, which is why initially we thought the minimum version would have to be Android 10.0 iirc but were able to lower it to 6.0.

I've seen other users ask for it to support Android 4.0.

I'll try to link the technical explanation we had here if that should be or any interest.

**Reply by zenseii** [reply]
Here is the technical explanation:
https://github.com/ihhub/fheroes2/issues/6008#issuecomment-1328128252

**Reply by zenseii** [reply]
To summarize, for the Android version you asked about we won't be able to support it due to SDL, a library we use, isn't supported below 4.1, in addition to other restrictions for things below 6.0 I'd imagine.

**Reply by oleg-derevenetz** [reply]
We can lower the requirements to Android 4.4 if there are people ready to test it and verify that fheroes2 really works on these devices. Android 4.4 KitKat was released about 9 years ago, so [not many devices are still working](https://www.androidauthority.com/wp-content/uploads/2022/05/android-distribution-chart-may-2022.jpg.webp).

**Reply by zenseii** [reply]
In fact I assume many of those legacy Android devices are old tablets people have lying around.

**Reply by rumly111** [reply]
thanks for the reply
